src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -132,4 +137,25 @@ private void validateRequestHeaders(String provenance, String sourceArtefactId)
             throw new EmptyRequestHeaderException(PublicationConfiguration.SOURCE_ARTEFACT_ID_HEADER);
         }
     }
+
+
+    /**
+     * Enforces conditional mandatory fields based on the publication type. This is to ensure that status updates are
+     * able to persist indefinitely if required.
+     */
+    public boolean validateDateFromDateTo(LocalDateTime displayFrom, LocalDateTime displayTo, ArtefactType type) {
+        if (type.equals(ArtefactType.LIST) || type.equals(ArtefactType.JUDGEMENT)
```

Comment:
this is a lot of ors for an if statement 😂  maybe an allowed and disallowed type to check or s cascading switch statement on the artefact types?

Reply:
hopefully fixed now - maybe it's a pretty ugly solution, but the switch statement was wayyyy too verbose considering the simplicity of the logic